### ISSUE TITLE
[BITV/A11Y] Zooming and Scaling to be turned onCannot control pdf viewer

### ISSUE THREAD

**Issue by AndyScherzinger** [nextcloud/files_pdfviewer#632]

**Comment by danxuliu** [comment]
`maximum-scale=1` was deliberately added to PDF.js and it is needed for proper display in mobile devices (see 7755 in https://github.com/mozilla/pdf.js/issues, not linking directly to avoid polluting the issue with backreferences). Nevertheless, the viewer provides the possibility to zoom in and out the text, and [since PDF.js 3.3.122 pinch-to-zoom is also supported](https://github.com/mozilla/pdf.js/commit/d6b7a1f034c1c6cb2458135a088633ae87310e1f)
